services/core-web/src/tests/components/search/__snapshots__/SearchResults.spec.js.snap

```diff
@@ -1,3 +1,62 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Search renders properly 1`] = `<Fragment />`;
+exports[`Search renders properly 1`] = `
+Object {
+  "asFragment": [Function],
+  "baseElement": <body>
+    <div />
+  </body>,
+  "container": <div />,
```

Comment:
just comparing this to other test results, would looking at container.firstChild be a bit better? Still an improvement over <Fragment />. 🤷‍♀️